wagtailio/static/sass/components/_card-grid.scss

```diff
@@ -0,0 +1,22 @@
+.card-grid {
+    margin-bottom: 20px;
+    display: flex;
+    flex-wrap: wrap;
+
+    @include media-query(medium) {
+        margin-bottom: 50px;
+    }
+
+    > div {
```

Comment:
Suggestion: This works, but i'd have a preference to using a class to make it explicit and avoid catching someone out if they where to add a `div` as a child. You could pass a class to `card_block.html` - something like:

`{% include "patterns/components/streamfields/cards/card_block.html" with value=card classes="card-grid__item" %}`